crates/swc_css_parser/tests/recovery/bad-url-token/double-quotes/output.json

```diff
@@ -88,49 +88,23 @@
         },
         "value": [
           {
-            "type": "ListOfComponentValues",
+            "type": "Declaration",
```

Comment:
Example of the better recovery mode - now we can parse it as declaration (due logic in spec)